### PR TITLE
Update Splash Screen 

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #
- Make loading logos extension point
- update styling ( use darkmode styling )  

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/rtk-query/ability.js [hunk 1]
```diff
@@ -60,5 +53,9 @@ export const LoadSessionGuard = ({ children }) => {
     );
   }
 
-  return children;
+  return (
+    <LoadingScreen isLoading={isLoading || !org?.id} message="Setting up your session">
```
Comment (aabidsofi19) [inline ui/rtk-query/ability.js:57]: @leecalcote  you might want to change the text here . this is show during loading org , setting up keys

Comment (leecalcote) [reply ui/rtk-query/ability.js:57]: Nice, do we have an array of possibilities?

Comment (leecalcote) [inline ui/rtk-query/ability.js:57]:
```suggestion
    <LoadingScreen isLoading={isLoading || !org?.id} message="This loading screen is temporary. Your YAML-induced suffering is eternal.">
```

File: ui/components/LoadingComponents/LoadingComponentServer.js [hunk 2]
```diff
@@ -14,11 +16,20 @@ function LoadingScreen(props) {
           placeItems: 'center',
           minHeight: '100vh',
           textAlign: 'center',
+          backgroundColor: '#202020',
```
Comment (leecalcote) [inline ui/components/LoadingComponents/LoadingComponentServer.js:19]:
```suggestion
          backgroundColor: '#263238', // color of navigation menu
```

File: ui/components/LoadingComponents/LoadingComponentServer.js [hunk 3]
```diff
@@ -14,11 +16,20 @@ function LoadingScreen(props) {
...
+          color: 'white',
```
Comment (leecalcote) [inline ui/components/LoadingComponents/LoadingComponentServer.js:20]:
```suggestion
          color: '#dedede', // soften the subtitle / message
```

**Review by jerensl — APPROVED** [review]

File: ui/rtk-query/ability.js (hunk 1 above)
Comment (aabidsofi19) [reply ui/rtk-query/ability.js]: not right now , but we could have similar setup . i didnt make it random choice so users know at what stage they are in . 
from loading app -> setting up session -> loading canvas .

**Pull request merged**